src/components/button/AccentColorButton.tsx

```diff
@@ -13,6 +14,10 @@ const AccentColorButton: React.FC<{
   className?: string;
   title?: string;
   noBg?: boolean;
+  disabledHint?: string;
+  onMouseEnter?: (e?) => void;
+  onMouseLeave?: (e?) => void;
+  onFocus?: (e?) => void;
```

Comment:
A bit YAGNI ... though I'm fine with having those

Reply:
True... The ant-design docs mentioned those events, but I just tested it without them and it works. Seems like it's not required when using a <div> wrapper around the button.